### ISSUE TITLE
BACs payment status support

### ISSUE THREAD

**Issue by judgej** [ardohr-ltd/starling-payments-objects#7]
Along with the `fasterPaymentStatus` objects returned for the payment account address, there is now a `bacsStatus` object with similar statuses.

A new PUT method allows the BACs statuses (diectCredit and diectDebit) to be enabled and disabled.

**Comment by judgej** [comment]
There are also half a dozen new BACs webhooks, and four new BACS endpoints (two GET and two PUT).

**Comment by bradydan** [comment]
Are these all new additions to the Starling payments API?

We don’t need to do anything with BACS

**Comment by judgej** [comment]
Yes, these are all new. I'm trying to keep up with any changes to the API, so we don't get any surprises.

It is something to be aware of. We may, for example, miss some incoming funding payments if we are not watching the BACS webhooks.

**Comment by bradydan** [comment]
We can’t accommodate changes at this stage. Otherwise it’s an ever-moving target. Make a note of the webhook/API changes but there should be no coding changes at this stage.
